cmd/minikube/cmd/delete.go

```diff
@@ -171,6 +198,16 @@ func runDelete(cmd *cobra.Command, args []string) {
 	delCtx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
 	defer cancel()
 
+	// get image names before containers deletion
+	dockerImageNames, err := kicbaseImages(delCtx, oci.Docker)
+	if err != nil {
+		klog.Warningf("error fetching docker images: %v", err)
+	}
+	podmanImageNames, _ := kicbaseImages(delCtx, oci.Podman)
```

Comment:
Skipping the error check here

Reply:
@spowelljr thanks for the review. Fixed.